test/redbird_test.exs

```diff
@@ -117,25 +107,33 @@ defmodule RedbirdTest do
     end
   end
 
-  test "redbird_session is appended to key names by default" do
-    conn = %{}
+  test "redbird_session is preprended to key names by default" do
+    conn = :get |> conn("/") |> sign_conn()
     options = []
     key = REDIS.put(conn, nil, %{foo: :bar}, options)
 
-    assert key =~ "redbird_session_"
+    assert key =~ ~r(\Aredbird_session_)
   end
 
   test "user can set their own key namespace" do
     Application.put_env(:redbird, :key_namespace, "test_")
 
-    Redbird.Redis.keys("test_*")
-    |> Redbird.Redis.del()
+    ensure_no_keys_with_prefix("test_")
 
-    conn = %{}
+    conn = :get |> conn("/") |> sign_conn()
     options = []
     key = REDIS.put(conn, nil, %{foo: :bar}, options)
 
-    assert key =~ "test_"
+    assert key =~ ~r(\Atest_)
     Application.delete_env(:redbird, :key_namespace)
   end
+
+  defp ensure_no_keys_with_prefix(prefix) do
+    prefix <> "*"
+    |> Redbird.Redis.keys()
+    |> Redbird.Redis.del()
+  end
+
+  # TODO: What happens with a non-verifiable key? Return an error? Return the empty result?
+  # test "non-verifable key ???"
```

Comment:
Is this something we still need to address or can this comment be removed?

Reply:
Removing. Thanks. Sorry for the sloppiness.